### DISCUSSION TITLE
Album selection

### DISCUSSION THREAD

**Discussion by Sonusss** [music-assistant/support#1449] [Feature requests and ideas]
Hi, 
I have plenty of Jazz files where almost every track is placed among a separate album depending of playing musicians. This makes a lot of albums with only one or two songs...

I suggest to have one of these tree options:
- A display value in the album icon showing track amount (but not very useful)
- A trigger value to validate an "Album" in term of number of tracks....
- A way to group "artists" or "albums" by partial match on artist names
All of these having a play all option

**Comment by OzGav** [comment]
If you use an automation you spare able to specify an artist as the thing to play.

**Reply by Sonusss** [reply]
I guess you will agree that your solution is not quite elegant and practical...

**Reply by Sonusss** [reply]
Ooooops, in the meantime I found the multiple selection option ...
